pkg/sdk/helper_test.go

```diff
@@ -19,14 +19,29 @@ func testClient(t *testing.T) *Client {
 
 func testSecondaryClient(t *testing.T) *Client {
 	t.Helper()
+	return testClient(t, testprofiles.Secondary)
+}
+
+func testThirdClient(t *testing.T) *Client {
```

Comment:
Nit: thirdTestClient, etc. to be consistent with profile names.

Reply:
Changed the names